crates/bevy_winit/src/lib.rs

```diff
@@ -147,6 +147,14 @@ pub fn winit_runner(mut app: App) {
 
     app.resources.insert_thread_local(event_loop.create_proxy());
 
+    // Create Windows and WinitWindows resources, so startup systems
+    // in below app.initialize() have access to them.
```

Comment:
I'd take out "in below app.initialize()" but that's not a big deal.

```
// Create Windows and WinitWindows resources so
// startup systems have access to them.
```

Reply:
The goal of this mention is so one do not move the call below app.initialize() as this would render it useless.